packages/next-swc/crates/next-core/src/next_import_map.rs

```diff
@@ -280,6 +280,20 @@ pub async fn get_next_server_import_map(
                     "next/dist/build/webpack/loaders/next-flight-loader/action-proxy",
                 ),
             );
+            import_map.insert_exact_alias(
+                "private-next-rsc-action-client-wrapper",
+                request_to_import_mapping(
+                    project_path,
+                    "next/dist/build/webpack/loaders/next-flight-loader/action-client-wrapper",
```

Comment:
The path for that file is a bit weird. Maybe we should move that out of the `next-flight-loader`

Reply:
Yeah we should do that at some point 👍